### DISCUSSION TITLE
Training Load Only Added For Last Ride

### DISCUSSION THREAD

**Discussion by iwoloschin** [philosowaffle/peloton-to-garmin#629] [TE / TSS / Vo2 / Intensity Minutes]
I did 3 rides yesterday, a 5 minute warmup, a 30 minute ride, and a 5 minute cooldown. After completing the cooldown I manually sync'd via Github Actions. All 3 rides show a Training Effect and calculate an Exercise Load, but only the 5 minute cooldown seems to have been added to the Acute Training Load chart.

Any thoughts on why this might be?

**Comment by iwoloschin** [comment]
After some testing over the past week it seems as if only the last ride will add to Training Load. If I do a Warmup, Ride, Cooldown and sync after each ride completes all rides count towards Training Load. Waiting to sync until all 3 rides are complete results in only the last ride counting towards Training Load.

**Reply by jayflan** [reply]
Has anyone had issues with training load not updating between non-consecutive rides?

Recorded and synced a ride yesterday, which shows training effect, but the load graph hasn't updated.  Also dropped my status to Recovery.  

A previous ride recorded two days ago, with similar load, recorded just fine and affected the training load graph. 

My status has also dropped, which usually isn't the case after the type of ride recorded yesterday.
